### PR TITLE
changed variable name buffer to jmsContext

### PR DESCRIPTION
**Pull request by lukasherman**
Changed misleading variable name `buffer` to `jmsContext`.

### CODE REVIEW

**Comment by eminn** [comment]
Hi Lukas, Thanks for your contribution.

Could you please sign the Hazelcast Contributor Agreement [1] ? 

[1] -> https://hazelcast.atlassian.net/wiki/spaces/COM/pages/6357071/Hazelcast+Contributor+Agreement

**Comment by viliam-durina** [comment]
verify

File: hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/connector/WriteJmsP.java [hunk 1]
```diff
@@ -20,19 +20,11 @@
...
-import com.hazelcast.jet.function.DistributedConsumer;
-import com.hazelcast.jet.function.DistributedFunction;
-import com.hazelcast.jet.function.DistributedSupplier;
-
-import javax.jms.Connection;
-import javax.jms.Destination;
-import javax.jms.Message;
-import javax.jms.MessageProducer;
-import javax.jms.Session;
+import com.hazelcast.jet.function.*;
```
Comment (cangencer) [inline hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/connector/WriteJmsP.java]: wildcard imports should be avoided

**Comment by viliam-durina** [comment]
@lukasherman, our style rules don't allow the star `*` imports, push a commit reverting the change in imports.

**Pull request merged**